site/docs/v1/tech/apis/scim/_scim-enterpriseuser-request-body.adoc

```diff
@@ -0,0 +1,72 @@
+
+- https://datatracker.ietf.org/doc/html/rfc7643#page-19
+
+==== Request Body
+[.api]
+[field]#active# [type]#[Boolean]# [required]#Required#::
+The state of the SCIM Resource.
+
+[field]#schemas# [type]#[Array]# [required]#Required#::
+The String identifier of the SCIM Resource schemas and extensions provided in the request. This should match supported schemas and extensions for the provisioning server.
+
+[field]#externalId# [type]#[String]# [optional]#Optional#::
+The externalId is a unique identifier created by the client that identifies the resource in the SCIM client's environment. It could be used as the FusionAuth resource id as well, but that is not a requirement of the SCIM specification. That would need to be done by the defined incoming lambda if desired.
+
+[field]#name.formatted# [type]#[String]# [optional]#Optional#::
+his would typically be the value used to populate the FusionAuth User.fullName by the incoming request lambda.
+
+[field]#name.familyName# [type]#[String]# [optional]#Optional#::
+his would typically be the value used to populate the FusionAuth User.lastName by the incoming request lambda.
+
+[field]#name.givenName# [type]#[String]# [optional]#Optional#::
+his would typically be the value used to populate the FusionAuth User.firstName by the incoming request lambda.
+
+[field]#name.middleName# [type]#[String]# [optional]#Optional#::
+his would typically be the value used to populate the FusionAuth User.middleName by the incoming request lambda.
+
+[field]#name.honorificPrefix# [type]#[String]# [optional]#Optional#::
+A prefix to be applied to the user's full name.
+
+[field]#name.honorificSuffix# [type]#[UUID]# [optional]#Optional#::
+A suffix to be applied to the user's full name.
+
+[field]#userName# [type]#[String]# [optional]#Optional#::
+This would typically be the value used to populate the FusionAuth User.userName by the incoming request lambda.
+
+[field]#password# [type]#[String]# [optional]#Optional#::
+This would typically be the value used to populate The FusionAuth User.password by the incoming request lambda.
+
+[field]#phoneNumbers# [type]#[Array]# [optional]#Optional#::
+SCIM supports multiple phone numbers while FusionAuth only allows a single phone number for a user. The incoming request lambda would need to determine which one to use if multiple are specified. The most common strategy would be to just use the one where primary is true.
+
+[field]#phoneNumbers[x].primary# [type]#[Boolean]# [optional]#Optional#::
+Indicates that this phone number is the user's primary phone number.
+
+[field]#phoneNumbers[x].value# [type]#[String]# [optional]#Optional#::
+This would typically be used to populate the FusionAuth User.mobilePhone by the incoming request lambda.
+
+[field]#phoneNumbers[x].type# [type]#[String]# [optional]#Optional#::
+This might be used to alternatively choose a value to decide which phone number to use to populate the FusionAuth User.mobilePhone by the incoming request lambda.
```

Comment:
```suggestion
The type of the phone number (work, home, mobile). Rather than use [field]#phoneNumbers[x].primary#, you could select a different number as the FusionAuth User.mobilePhone.
```